core/domain/state_domain.py

```diff
@@ -1062,12 +1061,12 @@ def traverse_by_schema_and_get(
             value_extractor: function. The function that extracts the wanted
                 value from each value that matches the obj_type's. It accepts
                 one parameter, the value that matches the search object type,
-                and returns a property of that value.
+                and optionally returns a property of that value.
```

Comment:
> I think you can drop this last clause ("and returns a property of that value"), because the function could be arbitrary, right?

>> Added "optionally" before "returns"

No, this is not what I mean. It may do a computation on some property and return the result of that. It may return the concatenation of two properties. It may do lots of different things, so the docstring as written is not accurate.